hip/test/solver/ir_kernels.cpp

```diff
@@ -175,7 +175,7 @@ TEST_F(Ir, ApplyWithIterativeInnerSolverIsEquivalentToRef)
 
     // Note: 1e-13 instead of 1e-14, as the difference in the inner gmres
     // iteration gets amplified by the difference in IR.
```

Comment:
```suggestion
    // Note: 1e-12 instead of 1e-14, as the difference in the inner gmres
    // iteration gets amplified by the difference in IR.
```